torch/cuda/__init__.py

```diff
@@ -82,7 +82,8 @@ def is_bf16_supported():
     r"""Returns a bool indicating if the current CUDA device supports dtype bfloat16"""
     cu_vers = torch.version.cuda
     if cu_vers is not None:
-        cuda_maj_decide = int(cu_vers.split(',')[0]) >= 11
+        cuda_maj_decide = int(cu_vers.split('.')[0]) >= 11
```

Comment:
I opened an issue for this here https://github.com/pytorch/pytorch/issues/64232. Is the ',' ever needed in some other locale, or was this just a bug?

Reply:
This exact change got it working for me, but I'm in the US so I wasn't sure if this was a battle of the decimal character or something :)

Reply:
No, just plain bug.